src/templates/pages/home.html

```diff
@@ -31,7 +33,22 @@ <h2>Accelerating reproducible computational research.</h2>
     {#                </div>#}
     {#            </div>#}
     {#        </div>#}
-
+    <script>
+    // Prevent buttons from stacking in mobile view
+    $(document).ready(function () {
+        $(window).resize(function resize() {
```

Comment:
```javascript
function resize() {
```

Can just be

```javascript
function {
```

Makes sense that it's an "anonymous" function since we're just passing it in here and not reusing it anywhere.